tests/test_dumps_escape.py

```diff
@@ -36,7 +36,7 @@
 lorem,b,'this is a valid statement with an \\% symbol',\'\\"b\\"\'
 lorem2,d,'this is a valid statement',\'\\"d\\"\'
 lorem3,c,'this is a valid statement with double quotes included \\"\\"\\"\\"\\"\\"\\"! ',\'\\"c\\"\'
-lorem4,a,'this is a valid statement with singlequotes included \\\' lol \\\'! ',\'\\"a\\"\'
```

Comment:
Basically, this should have been `"\\\\\\\'"` if it was encoding `"\\\'"`, not `"\\\'"`...

```py
>>> s = "\\\'"
>>> print(s)
\'
>>> print(arff.encode_string(s))
'\\\''
>>> arff.encode_string(s)
"'\\\\\\''"
```

Reply:
Do you mean there's an error here? I tested to read and write the following arff and the results are consistent with WEKA:
```
@RELATION string_escape

@ATTRIBUTE string STRING

@DATA

"\'"
"\\'"
"\\\'"
"\\\\'"
"\\\\\'"
"\\\\\\'"
```
Output:
```
@RELATION string_escape

@ATTRIBUTE string STRING

@DATA
'\''
'\\\''
'\\\''
'\\\\\''
'\\\\\''
'\\\\\\\''
```
WEKA
```
@relation string_escape

@attribute string string

@data
'\''
'\\\''
'\\\''
'\\\\\''
'\\\\\''
'\\\\\\\''
```

Reply:
Yes, there is an error in the functionality in master, required for this test to pass. Now the test and implementation are fixed, imo.